src/main/kotlin/com/zaxxer/ping/IcmpPinger.kt

```diff
@@ -333,11 +354,19 @@ class IcmpPinger(private val responseHandler:PingResponseHandler) {
          cc -= headerLen
 
          icmp.useMemory(socketBufferPointer.slice(headerLen.toLong()))
-         if (icmp.icmp_type.get() != ICMP_ECHOREPLY) {
-            LOGGER.fine({"   ^ Opps, not our response."})
-         }
-         else {
-            val seq = ntohs(icmp.icmp_hun.ih_idseq.icd_seq.shortValue())
+         icmp6.useMemory(socketBufferPointer)
```

Comment:
Is it true that we do not need to "slice()" the ``socketBufferPointer`` like we do for ``icmp`` (line 356)?